src/AzureAuth.Test/AzureAuth.Test.csproj

```diff
@@ -8,8 +8,7 @@
     <TargetFramework>net6.0</TargetFramework>
     <IsPackable>false</IsPackable>
 
-    <!-- Stylecop warnings as errors flag for build to fail -->
-    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>False</TreatWarningsAsErrors>

```

Comment:
This still shows the issues, and you can fix them - but doesn't constantly break Live unit testing. I think we should downgrade to warnings in all projects.